man/man8/zpool.8

```diff
@@ -1341,1228 +151,79 @@ or all properties if
-.It Xo
-.Nm
-.Cm upgrade
-.Op Fl V Ar version
-.Fl a Ns | Ns Ar pool Ns ...
-.Xc
-Enables all supported features on the given pool.
-Once this is done, the pool will no longer be accessible on systems that do not
-support feature flags.
-See
-.Xr zpool-features 5
-for details on compatibility with systems that support feature flags, but do not
-support all features enabled on the pool.
-.Bl -tag -width Ds
-.It Fl a
-Enables all supported features on all pools.
-.It Fl V Ar version
-Upgrade to the specified legacy version.
-If the
-.Fl V
-flag is specified, no features will be enabled on the pool.
-This option can only be used to increase the version number up to the last
-supported legacy version number.
-.El
-.It Xo
-.Nm
-.Cm version
-.Xc
-Displays the software version of the
-.Nm
-userland utility and the zfs kernel module.
-.It Xo
-.Nm
-.Cm wait
-.Op Fl Hp
-.Op Fl T Sy u Ns | Ns Sy d
-.Op Fl t Ar activity Ns Oo , Ns Ar activity Ns Oc Ns ...
-.Ar pool
-.Op Ar interval
-.Xc
+.It Xr zpool-wait 8
 Waits until all background activity of the given types has ceased in the given
 pool.
-The activity could cease because it has completed, or because it has been
-paused or canceled by a user, or because the pool has been exported or
-destroyed.
-If no activities are specified, the command waits until background activity of
-every type listed below has ceased.
-If there is no activity of the given types in progress, the command returns
-immediately.
-.Pp
-These are the possible values for
-.Ar activity ,
-along with what each one waits for:
-.Bd -literal
-        discard       Checkpoint to be discarded
-        free          'freeing' property to become 0
-        initialize    All initializations to cease
-        replace       All device replacements to cease
-        remove        Device removal to cease
-        resilver      Resilver to cease
-        scrub         Scrub to cease
-.Ed
-.Pp
-If an
-.Ar interval
-is provided, the amount of work remaining, in bytes, for each activity is
-printed every
-.Ar interval
-seconds.
+.El
+.Ss Fault Resolution
 .Bl -tag -width Ds
-.It Fl H
-Scripted mode.
-Do not display headers, and separate fields by a single tab instead of arbitrary
-space.
-.It Fl p
-Display numbers in parsable (exact) values.
-.It Fl T Sy u Ns | Ns Sy d
-Display a time stamp.
-Specify
-.Sy u
-for a printed representation of the internal representation of time.
-See
-.Xr time 2 .
-Specify
-.Sy d
-for standard date format.
-See
-.Xr date 1 .
+.It Xo
+.Xr zpool-offline 8
+.Xr zpool-online 8
+.Xc
+Takes the specified physical device offline or brings it online.
+.It Xr zpool-resilver 8
+Starts a resilver. If an existing resilver is already running it will be
+restarted from the beginning.
+.It Xr zpool-reopen 8
+Reopen all the vdevs associated with the pool.
+.It Xr zpool-clear 8
+Clears device errors in a pool.
 .El
+.Ss Import & Export
+.Bl -tag -width Ds
+.It Xr zpool-import 8
+Lists pools available to import.
```

Comment:
How about: `Make disks containing ZFS storage pools available for use on the system.`

Reply:
916a8b072